action.yml

```diff
@@ -56,15 +56,21 @@ inputs:
     default: "true"
     description: Whether or not to commit changes locally
 
+  tag:
+    type: string
+    required: false
+    default: "true"
```

Comment:
This should be true by default:
https://python-semantic-release.readthedocs.io/en/latest/commands.html#tag-no-tag